PractiseMay11/PlantsTask/Gallery.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PractiseMay11.PlantsTask
+{
+    internal class Gallery
+    {
+        public Plant[] plants;
+
+        public Worker worker;
+
+        public Gallery(Plant[] plants, Worker worker)
+        {
```

Comment:
I'd add validation here to check if plants is not null, if any plant exists and if worker is not null.